src/officehours_api/models.py

```diff
@@ -94,6 +94,7 @@ class Queue(SafeDeleteModel):
         models.CharField(max_length=20, choices=get_backend_types(), blank=False),
         default=get_default_allowed_backends,
     )
+    phys_location = models.CharField(max_length=100, blank=True)
```

Comment:
I like `inperson_location`, since it makes it crystal clear that it's directly tied to a particular backend. It's unfortunate that backend details are now bleeding into the database, but it probably wouldn't be worthwhile to pursue a cleaner but heavier solution, such as a JSON `backend_defaults`, unless we think it would get some use in the future. Are there any plans to have more backend-specific options at the Queue level? @mfldavidson

Reply:
Good thoughts, I also like `inperson_location`. We could create a `QueueBackendSettings` model that FKs to the `Queue` and includes the backend type and a JSON blob as fields -- or some such. Validation for the JSON plus lots of conditional queue data in the frontend might make things a bit messy. I understand that "In Person" is a backend, but maybe it's fundamentally different enough from the others that the backend name leaking into the data model here isn't catastrophic. Zoom may come and go, but In Person is sticking around (I hope at least 😭 ). Some more ideas to consider I guess.

Reply:
Yeah, I agree it's not a big code quality concession, so no complaints there.